client/web/compose/src/components/ModuleFields/Editor/Bool.vue

```diff
@@ -1,34 +1,46 @@
 <template>
   <b-form-group
-    label-class="d-flex align-items-center text-primary p-0"
+    :label-cols-sm="options.horizontalFieldLayoutEnabled && '5'"
+    :label-cols-lg="options.horizontalFieldLayoutEnabled && '4'"
+    :content-cols="options.horizontalFieldLayoutEnabled"
   >
     <template
-      v-if="field.options.switch"
+      v-if="field.options.switch || options.horizontalFieldLayoutEnabled"
```

Comment:
You dont need the extra check since the only thing that matters is the first condition